cmd/get.go

```diff
@@ -54,6 +54,24 @@ func getCMDsFlags(getCmd *cobra.Command) {
 	getCmd.PersistentFlags().Float64("min-space-required", 0, "Minimum space required in GB to continue the crawl. Default will be 50GB * (total disk space / 256GB) if total disk space is less than 256GB, else 50GB.")
 	getCmd.PersistentFlags().Bool("strict-regex", false, "If turned on, the xurls `strict` regex setting will be used. Otherwise a looser regex will be used.")
 
+	// Headless flags
+	getCmd.PersistentFlags().Bool("headless", false, "[headless] Run in headless mode (experimental)")
+	getCmd.PersistentFlags().Bool("headfull", false, "[headless] Run in headfull mode")
```

Comment:
Perhaps a small explanation for what 'headfull' actually does in case it isn't obvious?

Reply:
71c47bbd25c651119d189640698cd08ec1bd76be